egs/csj/s5/local/csj_train_lms.sh

```diff
@@ -50,9 +50,9 @@ done
 loc=`which ngram-count`;
 if [ -z $loc ]; then
   if uname -a | grep 64 >/dev/null; then # some kind of 64 bit...
-    sdir=`pwd`/../../../tools/srilm/bin/i686-m64 
+    sdir=${KALDI_ROOT:-`pwd`/../../..}/tools/srilm/bin/i686-m64 
```

Comment:
i think it would make more sense to have the script source path.sh
(`. ./path.sh`)
and then jsut use ${KALDI_ROOT} with no backoff to ../../../.

Reply:
Thank you for your advice.
Looks good to me.